server/src/test/java/org/elasticsearch/indices/analysis/AnalysisModuleTests.java

```diff
@@ -425,12 +437,17 @@ public void testRegisterHunspellDictionary() throws Exception {
 
     // Simple char filter that appends text to the term
     public static class AppendCharFilter extends CharFilter {
-        private final char[] appendMe;
-        private int offsetInAppendMe = -1;
+
+        static Reader append(Reader input, String appendMe) {
```

Comment:
I assume this is an optimization unrelated to the tokenizer move? Can you quickly confirm?

Reply:
It isn't so much an optimization as "this should be easier to read". I think.